benchmarks/accuracy/README.md

```diff
@@ -0,0 +1,99 @@
+# Accuracy Benchmark
+
+## Overview
+
+The Accuracy Benchmark evaluates the performance of TorchAcc using [FastChat](https://github.com/AlibabaPAI/FastChat_TorchAcc) against a baseline established by Torch native. The benchmark aims to ensure that TorchAcc maintains comparable accuracy levels with Torch native.
+
+## Evaluation Process
+
+To perform the evaluation, follow these steps:
+
+1. Set Baseline
+
+    ```bash
+    bash ./llama.sh <ORIGINAL_MODEL_DIR> 0
+    ```
+
+    Run the Torch native job using `run_clm.py`, a script copied from HuggingFace Transformers.
+
+2. Run TorchAcc
+
+    ```bash
+    bash ./llama.sh <ORIGINAL_MODEL_DIR> 1
+    ```
+
+    Run the TorchAcc job using the same script as used for Torch native.
+
+3. Evaluate Original
+
+    ```bash
+    bash ./mtbench.sh <ORIGINAL_MODEL_DIR>
+    ```
+
+    Evaluate the original checkpoint using FastChat.
+
+4. Evaluate Outputs
+
+    ```bash
+    bash ./mtbench.sh <TORCH_NATIVE_CHECKPOINT>
+    bash ./mtbench.sh <TORCHACC_CHECKPOINT>
+    ```
+
+    Evaluate the checkpoints output by Torch native job and TorchAcc.
+
+5. Compare Results
+
+    Compare the training and evaluation results.
+
+
+You can simply execute the `run.sh` script to perform all the steps.
+
+## Main Files
+
+* run.sh
+
+    The script integrates all the steps.
+
+    ```bash
+    bash ./run.sh <local_model_dir>
```

Comment:
What's difference between `ORIGINAL_MODEL_DIR` and `local_model_dir`?

Reply:
In the Evaluation Process, ORIGINAL_MODEL_DIR refers to the checkpoint downloaded from HuggingFace or ModelScope. This name is used to differentiate between TORCH_NATIVE_CHECKPOINT and TORCHACC_CHECKPOINT, making the evaluation process more clearly. The local_model_dir mentioned here allows you to specify any checkpoint path. This explanation clarifies the usage of the bash script.

Reply:
Maybe I missed it, it seems that when users don't specify checkpoint_output_dir but provides the correct number of parameters, the Bash log won't output where the checkpoint is written to. We need to get the source code to know parameters' means for sure?

Reply:
Using run.sh, users do not need to know the output checkpoint path. I will add some explanation in the documentation.

Reply:
done